fms/models/roberta.py

```diff
@@ -325,6 +323,30 @@ def _hf_sd_to_fms_sd(hf_sd: Mapping[Any, Any]) -> Mapping[Any, Any]:
         if name == "roberta.embeddings.position_embeddings.weight":
             new_sd[new_name] = new_sd[new_name][2:]
 
+        if (
+            "attn.query" in new_name
+            or "attn.key" in new_name
+            or "attn.value" in new_name
+        ):
+            unfused_weights = [
+                re.sub(
+                    r"attention\.self\.(key|query|value)", "attention.self.query", name
+                ),
+                re.sub(
+                    r"attention\.self\.(key|query|value)", "attention.self.key", name
```

Comment:
why is this `attention.self.*` ?

Reply:
Looks like roberta from huggingface is doing the following:

```python
self.self = RobertaSelfAttention(config, position_embedding_type=position_embedding_type)
```

So this appears to be correct